optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/resources/dev-templates/model.html

```diff
@@ -0,0 +1,74 @@
+<!--
+~ Copyright 2021 Red Hat, Inc. and/or its affiliates.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~      http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+{#include main}
+
+{#title}Model{/title}
+
+<!--
+  ~ Copyright 2021 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+{#body}
+<div class="card h-100 shadow">
+    <div class="card-header bg-primary text-light">Solution: {info:solverConfigProperties.optaPlannerModelProperties.solutionClass}</div>
+    <div class="card-body">
+    {#for entityClass in info:solverConfigProperties.optaPlannerModelProperties.entityClassList}
+    <div class="card h-100 shadow">
+        <div class="card-header">Entity: {entityClass}</div>
+        <div class="card-body">
+            <table class="table table-striped">
+                <thead class="thead-dark">
+                    <tr>
+                        <th scope="col">Genuine Variables</th>
+                    </tr>
+                </thead>
+                <tbody>
+                {#for genuineVariable in info:solverConfigProperties.optaPlannerModelProperties.entityClassToGenuineVariableListMap.get(entityClass)}
+                    <tr><td>{genuineVariable}</td></tr>
+                {/for}
+                </tbody>
+            </table>
+            <table class="table table-striped">
+                <thead class="thead-dark">
+                <tr>
+                    <th scope="col">Shadow Variables</th>
+                </tr>
+                </thead>
+                <tbody>
+                {#for shadowVariable in info:solverConfigProperties.optaPlannerModelProperties.entityClassToShadowVariableListMap.get(entityClass)}
+                    <tr><td>{shadowVariable}</td></tr>
+                {/for}
+                </tbody>
+            </table>
+        </div>
+    </div>
```

Comment:
We should either hide the entire section if there are no genuine/shadow vars, or better yet say "None."
Leaving the section there empty looks bad.

Reply:
Removed the section if it empty (it also possible to do "None", but I think it wastes space).